imports/plugins/included/product-variant/components/productGrid.js

```diff
@@ -81,6 +82,26 @@ class ProductGrid extends Component {
     }
   }
 
+  renderFilteredCount() {
+    const { selectedProductIds } = this.props;
+    const count = selectedProductIds.length;
+    const filterByProductIds = Session.get("filterByProductIds");
+
+    if (filterByProductIds) {
+      return (
+        <div style={{ paddingTop: "24px", paddingLeft: "8px" }}>
+          <Typography variant="h4" style={{ display: "inline" }}>
```

Comment:
MUI's `Typography` component has a built in prop for inline display, the `display` prop. You can use that here instead of `display: "inline"` in a `style` tag.

https://material-ui.com/api/typography/